src/cli.rs

```diff
@@ -308,6 +372,42 @@ pub fn make_cli_subcommands<'a, 'b>() -> App<'a, 'b> {
                 )
 }
 
+/// Add [clap](https://docs.rs/clap/) global wallet options to an `App`
+///
+/// Additional app specific options can be added. See [clap](https://docs.rs/clap/) for more
+/// information on how to customize the global flags for an `App`.
+///
+/// # Wallet global flags
+///
+/// | Short | Long  | Description |
+/// |-------|-------|-------------|
+/// | -c | --change_descriptor <DESCRIPTOR>            | Sets the descriptor to use for internal addresses |
+/// | -d | --descriptor <DESCRIPTOR>                   | Sets the descriptor to use for the external addresses |
+/// | -e | --esplora <ESPLORA>                         | Use the esplora server if given as parameter |
+/// |    | --esplora_concurrency <ESPLORA_CONCURRENCY> | Concurrency of requests made to the esplora server [default: 4] |
+/// | -n | --network <NETWORK>                         | Sets the network [default: testnet]  [possible values: testnet, regtest] |
+/// | -p | --proxy <SERVER:PORT>                       | Sets the SOCKS5 proxy for the Electrum client |
+/// | -s | --server <SERVER:PORT>                      | Sets the Electrum server to use [default: ssl://electrum.blockstream.info:60002] |
+/// | -w | --wallet <WALLET_NAME>                      | Selects the wallet to use [default: main] |
+///
+/// # Example
+///
+/// ```rust
+/// let app = cli::make_cli_subcommands();
+/// let app = cli::add_global_flags(app);
+///
+/// let app = app.arg(
+///             Arg::with_name("custom")
+///                 .short("c")
+///                 .long("custom")
+///                 .value_name("CUSTOM")
+///                 .help("Sets the custom option")
+///                 .takes_value(true)
+///                 .default_value("option1")
+///                 .possible_values(&["option1", "option2"]),
+///         );
+/// ```
```

Comment:
Indentation a little weird here, too 
![image](https://user-images.githubusercontent.com/4335621/100286898-3ee88800-2f39-11eb-87cf-6bc1c7ad85d5.png)

Reply:
So our docstrings are tested to make sure they compile, but seems like `cargo-fmt` ignores them? That's gotta be configurable ...

Reply:
[rustfmt has an unstable feature to format comments](https://github.com/rust-lang/rustfmt/issues/3348)

Reply:
very cool! seems to work, here's the full command:
```shell
cargo +nightly fmt -- --config format_code_in_doc_comments=true
```